lib/twirp/version.rb

```diff
@@ -1,3 +1,3 @@
 module Twirp
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```

Comment:
I would say, hold off on the version bump until the client is working and has at least a few tests. We can merge this in master and make a `0.2.0` release PR when we are ready.